src/client/js/reportStore.js

```diff
@@ -51,6 +51,16 @@ class ReportStore {
     this[prop] = !this[prop];
   }
 
+  @action.bound
+  filterOnly(prop) {
```

Comment:
The way this currently works is that once you've clicked a quick summary button to filter the results, you cannot quickly go back to the initial setting. In addition, there are cases where the toggles in the side menu are disabled, for example when you have no pending tests. This feature should account for that.